src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/RefVsAnyResult.java

```diff
@@ -0,0 +1,69 @@
+package org.broadinstitute.hellbender.tools.walkers.haplotypecaller;
+
+/**
+ * Holds information about a genotype call of a single sample reference vs. any non-ref event
+ */
+final class RefVsAnyResult {
+    /**
+     * The genotype likelihoods for ref/ref ref/non-ref non-ref/non-ref
+     */
+    private final double[] genotypeLikelihoods;
+
+    /**
+     * AD field value for ref / non-ref
+     */
+    private final int[] AD_Ref_Any = new int[2];
```

Comment:
Actually I would split it in to ```int``` variables. It seems that get_AD_Ref_Any() bellow is used in such a restricted manner that is better just to create a in[] array on demand down there rather that maintain the 2 position array as a field. So:

```
 private int refDepth = 0;
 private int nonRefDepth = 0;
```